src/main/app/src/tools/utils.ts

```diff
@@ -320,3 +321,23 @@ export const createOsaamismerkinKuvausHtml = (
     )
   );
 };
+
+export function reloadPage() {
+  // Tarvitaan uudelleenlatausta että Matomo tilastojen
+  // seurannan skripti suoriutuu index.html:ssa
+  window.location.replace(window.location.href);
+}
+
+export function setMandatoryCookieAccepted() {
+  Cookies.set(COOKIES.MANDATORY_COOKIE_NAME, 'true', {
+    expires: 1800,
+    path: '/',
+  });
+}
+
+export function setStatisticsCookiesAccepted() {
+  Cookies.set(COOKIES.STATISTICS_COOKIE_NAME, 'true', {
+    expires: 1800,
+    path: '/',
+  });
+}
```

Comment:
Nämä persistoinnit voisi tosiaan siirtää tuonne useCookiesInfo-hookkiin. Voiko tässä muuten nyt ollenkaan poistaa näitä keksejä? Kai tässä oli tarkoituksena se, että pystyi muokkaamaan näitä asetuksia?